annotation-processor/src/main/java/org/hibernate/validator/ap/classchecks/AbstractClassCheck.java

```diff
@@ -20,15 +22,15 @@
  * supported element types.
  * </p>
  * <p>
- * All check methods not overridden will return an empty list.
+ * All check methods not overridden will return an empty set.
  * </p>
  *
  * @author Marko Bekhta
  */
 public abstract class AbstractClassCheck implements ClassCheck {
 
 	@Override
-	public Collection<ConstraintCheckIssue> checkMethod(ExecutableElement element) {
+	public Set<ConstraintCheckIssue> checkMethod(ExecutableElement element) {
```

Comment:
Yes, it's just for the sake of consistency. I must admit I wouldn't have changed it if I didn't have other changes to do.

To be honest, if I had to do it from the ground up, I think I would have used ArrayList everywhere to have the result as predictable as possible. I don't think Sets are of much value here.